SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKEditorSettings.h

```diff
@@ -130,9 +85,13 @@ struct FWorkerTypeLaunchSection
 	{
 	}
 
-	/** Worker type name, deprecated in favor of defining them in the runtime settings.*/
-	UPROPERTY(config)
-	FName WorkerTypeName_DEPRECATED;
+	/** Worker type name. */
+	UPROPERTY(Category = "SpatialGDK", EditAnywhere, config)
+	FName WorkerTypeName;
```

Comment:
Is this used?

Reply:
Yup, WorkerTypes are back in v15 and must be included in the launch config.